run/django/e2e_test.py

```diff
@@ -0,0 +1,412 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This test creates a Cloud SQL instance, a Cloud Storage bucket, associated
+# secrets, and deploys a Django service
+
+import os
+import subprocess
+import uuid
+
+from google.cloud import secretmanager_v1 as sm
+import pytest
+import requests
+
+# Unique suffix to create distinct service names
+SUFFIX = uuid.uuid4().hex[:10]
+
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+REGION = "us-central1"
+
+CLOUD_STORAGE_BUCKET = f"{PROJECT}-media-{SUFFIX}"
+
+CLOUDSQL_INSTANCE = f"instance-{SUFFIX}"
```

Comment:
You should be able to access the instance via env var, `POSTGRES_INSTANCE`,  (https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.kokoro/tests/run_tests.sh#L113)